### DISCUSSION TITLE
More ESLint properties

### DISCUSSION THREAD

**Discussion by brettz9** [sapegin/mrm#307] [Ideas]
Hi,

Would you be open to a PR to add the remaining standard ESLint properties (of your `config` type):
"env", "globals", "ignorePatterns", "noInlineConfig", "overrides", "parser", "parserOptions", "plugins", "root", "settings"

I suppose they could either be separate options, or just all part of one extra `eslintConfig` object.

I'd also like to see about adding support for comma-separated `eslintPreset` (for `extends`).

**Comment by sapegin** [comment]
> I suppose they could either be separate options, or just all part of one extra eslintConfig object.

I guess this would be a better API than having a separate option for each property.

**Comment by brettz9** [comment]
Sure. But do you also want a breaking change to drop (or deprecate) `eslintRules` (in favor of `eslintConfig.rules`)?

(I am more certain you wouldn't want to fold `eslintPreset` into `eslintConfig.extends`, no less because that config, unlike `eslintRules` can be interactive.)

**Comment by sapegin** [comment]
> But do you also want a breaking change to drop (or deprecate) eslintRules (in favor of eslintConfig.rules)?

I think a breaking change for a better API is fine here ;-)
